src/tests/integration/test_duckDB.py

```diff
@@ -0,0 +1,15 @@
+def test_auto_commit_mode_on(ip_with_duckDB, capsys):
+    ip_with_duckDB.run_cell("%config SqlMagic.autocommit=True")
+    ip_with_duckDB.run_cell("%sql CREATE TABLE weather (city VARCHAR,);")
+    out, _ = capsys.readouterr()
+    assert "The database driver doesn't support such AUTOCOMMIT execution option" in out
+
+
+def test_auto_commit_mode_off(ip_with_duckDB, capsys):
+    ip_with_duckDB.run_cell("%config SqlMagic.autocommit=False")
+    ip_with_duckDB.run_cell("%sql CREATE TABLE weather (city VARCHAR,);")
+    out, _ = capsys.readouterr()
+    assert (
```

Comment:
better to check for the output of the confirmation message instead of looking that the error message doesn't exist